sdk/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -18,22 +18,33 @@
 
 import io.opentelemetry.OpenTelemetry;
 import io.opentelemetry.internal.Obfuscated;
+import io.opentelemetry.internal.Utils;
 import io.opentelemetry.sdk.correlationcontext.CorrelationContextManagerSdk;
+import io.opentelemetry.sdk.errorhandler.DefaultErrorHandler;
+import io.opentelemetry.sdk.errorhandler.ErrorHandler;
+import io.opentelemetry.sdk.errorhandler.OpenTelemetryException;
 import io.opentelemetry.sdk.metrics.MeterSdkProvider;
 import io.opentelemetry.sdk.trace.TracerSdkProvider;
+import javax.annotation.Nullable;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
  * This class provides a static global accessor for SDK telemetry objects {@link TracerSdkProvider},
- * {@link MeterSdkProvider} and {@link CorrelationContextManagerSdk}.
+ * {@link MeterSdkProvider} and {@link CorrelationContextManagerSdk}. It serves as a convenience
+ * class getting and casting the telemetry objects from {@link OpenTelemetry}.
  *
- * <p>This is a convenience class getting and casting the telemetry objects from {@link
- * OpenTelemetry}.
+ * <p>This class also provides static methods for accessing and setting a custom global error
+ * handler for the OpenTelemetry SDK.
  *
  * @see OpenTelemetry
  */
 @ThreadSafe
 public final class OpenTelemetrySdk {
+  private static final Object mutex = new Object();
+
+  @Nullable private static volatile OpenTelemetrySdk instance;
```

Comment:
rather than introducing a whole instance and double-checked locking in the getInstance, why not just have the errorHandler be static, since it's global anyway?

Reply:
+1 for the handler being static (we don't need anything fancy here, at least *for now*).